src/services/ExposureNotificationService/ExposureNotificationServiceProvider.tsx

```diff
@@ -8,9 +8,10 @@ import SystemSetting from 'react-native-system-setting';
 import {ContagiousDateInfo} from 'shared/DataSharing';
 import {useStorage} from 'services/StorageService';
 import {log} from 'shared/logging/config';
-import {EventTypeMetric, FilteredMetricsService} from 'services/MetricsService/FilteredMetricsService';
+import {FilteredMetricsService} from 'services/MetricsService/FilteredMetricsService';
 import {checkNotifications} from 'react-native-permissions';
 import {Status} from 'screens/home/components/NotificationPermissionStatus';
+import {EventTypeMetric} from 'services/MetricsService/MetricsFilter';
```

Comment:
Pending comment above this would be moved up and into a single import

Reply:
Done